### ISSUE TITLE
Lack of important context on git errors

### ISSUE THREAD

**Issue by adam-rh-se** [chrisdsa/Colligo#1]
When syncing, colligo does not display important context (ex: directory) when displaying errors from git. This may make diagnosing more difficult than necessary.

```
% colligo --sync
Parsing manifest file: manifest.xml
Synchronize all projects
ERROR [colligo::git_version_control] [some/project] error: Your local changes to the following files would be overwritten by checkout:
        somefile.h
Please commit your changes or stash them before you switch branches.
Aborting

Error: CheckoutFailed
```

We can see which project failed, but we cannot directly know the corresponding directory, except by looking into the manifest file.

**Comment by chrisdsa** [comment]
This will be resolved in the next release (v0.3.0).

**Comment by chrisdsa** [comment]
Fixed in v0.3.0
